flask/app.py

```diff
@@ -157,6 +157,11 @@ class Flask(_PackageBoundObject):
     #: :class:`~flask.Response` for more information.
     response_class = Response
 
+    #: The class that is used for the Jinja environment.
+    #:
+    #: .. versionadded:: 0.11
```

Comment:
Hm I guess for now the different versions don't really matter as long as it's cleaned up before whatever the next release's version number will be ;)